sdk/communication/AzureCommunicationChat/Package.swift

```diff
@@ -24,7 +24,7 @@ let package = Package(
     dependencies: [
         .package(
             url: "https://github.com/Azure/azure-sdk-for-ios.git",
-            .revision("3e3c80d60173613c8dd4cb6b219188cf5070e8e7")
+            .branch("dev/AzureCore")
```

Comment:
In fact, I didn't realize this file existed. Do we need it? It seems like the repo should have only a single Package.swift, located in the root.

Reply:
It is generated and would be used in the case where you generated an SDK to live in your own repo. So in that sense, I suppose it does make sense to retain this since in that instance, the SDK would not be packaged with AzureCore.